packages/map-template/src/index.jsx

```diff
@@ -1,12 +1,43 @@
+import * as Sentry from "@sentry/react";
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM, { createRoutesFromChildren, matchRoutes, useLocation, useNavigationType } from 'react-dom/client';
 import App from './App';
 import './index.css';
 import reportWebVitals from './reportWebVitals';
 
+Sentry.init({
+    dsn: "https://0ee7fa162023d958c96db25e99c8ff6c@o351128.ingest.sentry.io/4506851619831808",
```

Comment:
Is this a secret that is now no longer a secret? 🙃 Should it be configurable for customers, or is it something that we own and manage 100%?

Reply:
@ammapspeople It's not a secret. Sentry writes [in their docs](https://docs.sentry.io/product/sentry-basics/concepts/dsn-explainer/#dsn-utilization):

> DSNs are safe to keep public because they only allow submission of new events and related event data; they do not allow read access to any information.
>
> While there is a risk of abusing a DSN, where any user can send events to your organization with any information they want, this is a rare occurrence. Sentry provides controls to [block IPs](https://docs.sentry.io/platform-redirect/?next=/configuration/options/) and similar concerns. You can also rotate (and revoke) DSNs by navigating to [Project] > Settings > Client Keys (DSN).

I don't see this as something that should be configurable for any other customers. Forking and customizing is the preferred option in that case.